2024-03-21 05:42:26,940 INFO    MainThread:75780 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-03-21 05:42:26,940 INFO    MainThread:75780 [wandb_setup.py:_flush():76] Configure stats pid to 75780
2024-03-21 05:42:26,940 INFO    MainThread:75780 [wandb_setup.py:_flush():76] Loading settings from C:\Users\CYP\.config\wandb\settings
2024-03-21 05:42:26,940 INFO    MainThread:75780 [wandb_setup.py:_flush():76] Loading settings from c:\Users\CYP\Documents\GitHub\AIAKI\wandb\settings
2024-03-21 05:42:26,940 INFO    MainThread:75780 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-03-21 05:42:26,940 INFO    MainThread:75780 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-03-21 05:42:26,942 INFO    MainThread:75780 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-03-21 05:42:26,942 INFO    MainThread:75780 [wandb_init.py:_log_setup():526] Logging user logs to c:\Users\CYP\Documents\GitHub\AIAKI\wandb\run-20240321_054226-hxvs1n67\logs\debug.log
2024-03-21 05:42:26,942 INFO    MainThread:75780 [wandb_init.py:_log_setup():527] Logging internal logs to c:\Users\CYP\Documents\GitHub\AIAKI\wandb\run-20240321_054226-hxvs1n67\logs\debug-internal.log
2024-03-21 05:42:26,942 INFO    MainThread:75780 [wandb_init.py:init():566] calling init triggers
2024-03-21 05:42:26,942 INFO    MainThread:75780 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'batch_size': 64, 'n_units': 16, 'learning_rate': 1e-06, 'lr_decay': 0.8, 'lr_step': 10, 'ns': 8, 'loss': 'mse', 'update_freq': 16}
2024-03-21 05:42:26,942 INFO    MainThread:75780 [wandb_init.py:init():591] re-initializing run, found existing run on stack: dsb9zdhi
2024-03-21 05:42:26,943 INFO    MainThread:75780 [wandb_run.py:_finish():1970] finishing run dahs/IMV_LSTM_AKI_new/dsb9zdhi
2024-03-21 05:42:26,944 ERROR   MainThread:75780 [jupyter.py:save_history():437] Run pip install nbformat to save notebook history
2024-03-21 05:42:26,944 INFO    MainThread:75780 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-03-21 05:42:26,944 INFO    MainThread:75780 [wandb_init.py:_jupyter_teardown():454] cleaning up jupyter logic
2024-03-21 05:42:26,944 INFO    MainThread:75780 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-03-21 05:42:26,944 INFO    MainThread:75780 [wandb_run.py:_restore():2193] restore
2024-03-21 05:42:26,945 INFO    MainThread:75780 [wandb_run.py:_restore():2199] restore done
2024-03-21 05:42:31,629 INFO    MainThread:75780 [wandb_run.py:_footer_history_summary_info():3866] rendering history
2024-03-21 05:42:31,631 INFO    MainThread:75780 [wandb_run.py:_footer_history_summary_info():3898] rendering summary
2024-03-21 05:42:31,637 INFO    MainThread:75780 [wandb_run.py:_footer_sync_info():3825] logging synced files
2024-03-21 05:42:31,694 INFO    MainThread:75780 [wandb_init.py:init():616] starting backend
2024-03-21 05:42:31,694 INFO    MainThread:75780 [wandb_init.py:init():620] setting up manager
2024-03-21 05:42:31,698 INFO    MainThread:75780 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-03-21 05:42:31,699 INFO    MainThread:75780 [wandb_init.py:init():628] backend started and connected
2024-03-21 05:42:31,703 INFO    MainThread:75780 [wandb_run.py:_label_probe_notebook():1295] probe notebook
2024-03-21 05:42:31,703 INFO    MainThread:75780 [wandb_run.py:_label_probe_notebook():1305] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-03-21 05:42:31,703 INFO    MainThread:75780 [wandb_init.py:init():720] updated telemetry
2024-03-21 05:42:31,727 INFO    MainThread:75780 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-03-21 05:42:33,171 INFO    MainThread:75780 [wandb_run.py:_on_init():2262] communicating current version
2024-03-21 05:42:33,262 INFO    MainThread:75780 [wandb_run.py:_on_init():2271] got version response upgrade_message: "wandb version 0.16.4 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-03-21 05:42:33,262 INFO    MainThread:75780 [wandb_init.py:init():804] starting run threads in backend
2024-03-21 05:42:33,373 INFO    MainThread:75780 [wandb_run.py:_console_start():2241] atexit reg
2024-03-21 05:42:33,373 INFO    MainThread:75780 [wandb_run.py:_redirect():2096] redirect: wrap_raw
2024-03-21 05:42:33,373 INFO    MainThread:75780 [wandb_run.py:_redirect():2161] Wrapping output streams.
2024-03-21 05:42:33,373 INFO    MainThread:75780 [wandb_run.py:_redirect():2186] Redirects installed.
2024-03-21 05:42:33,374 INFO    MainThread:75780 [wandb_init.py:init():847] run started, returning control to user process
2024-03-21 06:27:14,658 INFO    MainThread:75780 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-03-21 06:27:14,658 INFO    MainThread:75780 [wandb_setup.py:_flush():76] Configure stats pid to 75780
2024-03-21 06:27:14,658 INFO    MainThread:75780 [wandb_setup.py:_flush():76] Loading settings from C:\Users\CYP\.config\wandb\settings
2024-03-21 06:27:14,658 INFO    MainThread:75780 [wandb_setup.py:_flush():76] Loading settings from c:\Users\CYP\Documents\GitHub\AIAKI\wandb\settings
2024-03-21 06:27:14,658 INFO    MainThread:75780 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-03-21 06:27:14,658 INFO    MainThread:75780 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-03-21 06:27:14,659 INFO    MainThread:75780 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-03-21 06:27:14,659 INFO    MainThread:75780 [wandb_init.py:_log_setup():526] Logging user logs to c:\Users\CYP\Documents\GitHub\AIAKI\wandb\run-20240321_062714-tuf04jbl\logs\debug.log
2024-03-21 06:27:14,659 INFO    MainThread:75780 [wandb_init.py:_log_setup():527] Logging internal logs to c:\Users\CYP\Documents\GitHub\AIAKI\wandb\run-20240321_062714-tuf04jbl\logs\debug-internal.log
2024-03-21 06:27:14,659 INFO    MainThread:75780 [wandb_init.py:_jupyter_setup():472] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001FB34599E10>
2024-03-21 06:27:14,659 INFO    MainThread:75780 [wandb_init.py:init():566] calling init triggers
2024-03-21 06:27:14,659 INFO    MainThread:75780 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'batch_size': 64, 'n_units': 2, 'learning_rate': 1e-06, 'lr_decay': 0.8, 'lr_step': 10, 'ns': 8, 'loss': 'mse', 'update_freq': 16}
2024-03-21 06:27:14,659 INFO    MainThread:75780 [wandb_init.py:init():591] re-initializing run, found existing run on stack: hxvs1n67
2024-03-21 06:27:14,660 INFO    MainThread:75780 [wandb_run.py:_finish():1970] finishing run dahs/IMV_LSTM_AKI_new/hxvs1n67
2024-03-21 06:27:14,660 INFO    MainThread:75780 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-03-21 06:27:14,661 INFO    MainThread:75780 [wandb_run.py:_restore():2193] restore
2024-03-21 06:27:14,661 INFO    MainThread:75780 [wandb_run.py:_restore():2199] restore done
2024-03-21 06:27:18,042 INFO    MainThread:75780 [wandb_run.py:_footer_history_summary_info():3866] rendering history
2024-03-21 06:27:18,043 INFO    MainThread:75780 [wandb_run.py:_footer_history_summary_info():3898] rendering summary
2024-03-21 06:27:18,050 INFO    MainThread:75780 [wandb_run.py:_footer_sync_info():3825] logging synced files
